### PR TITLE
feat(mwan): added object support to mwan

### PR DESCRIPTION
**Pull request by Tbaile**
Adding support for objects in MWAN rules

Ref:
- Issue: https://github.com/NethServer/nethsecurity/issues/671
- Merge after: https://github.com/NethServer/nethsecurity/pull/674

---

Images:
image
image
image

### CODE REVIEW

File: src/components/standalone/users_objects/ObjectUsages.vue [hunk 1]
```diff
@@ -85,7 +85,8 @@ function getManagementPageLabel(category: string) {
       return t('common.go_to_page', { page: t('standalone.firewall_rules.title') })
     case 'objects':
       return t('common.go_to_page', { page: t('standalone.objects.title') })
-    //// add other pages
+    case 'mwan3':
```
Comment (andre8244) [inline src/components/standalone/users_objects/ObjectUsages.vue:88]: I'd leave `//// add other pages`, we'll need it for port forward

File: src/components/standalone/users_objects/ObjectUsages.vue [hunk 2]
```diff
@@ -98,7 +99,9 @@ function goToManagementPage(subtype: string) {
       break
     case 'objects':
       path = 'users-objects/objects'
-    //// add other pages
+      break
```
Comment (andre8244) [inline src/components/standalone/users_objects/ObjectUsages.vue:102]: Same as above, I'd leave `//// add other pages`, we'll need it for port forward

**Review by andre8244 — COMMENTED** [review]
- Objects page is left untouched. For the sake of maintenance, if we adopt this new implementation I suggest to use it everywhere
- Object are never reloaded. I'd do that in Objects page:
  - on page load
  - on object creation/modification

**Comment by Tbaile** [comment]
> * Objects page is left untouched. For the sake of maintenance, if we adopt this new implementation I suggest to use it everywhere

This will drastically add more time to fix due to refactoring everything, not sure if it's better to leave the deprecation warnings and then refactor it when additional changes are needed. Otherwise more (probably much) work is needed.

> * Object are never reloaded. I'd do that in Objects page:
>   
>   * on page load
>   * on object creation/modification

Once the store loads into the app, the fetch is automatic. Adding refresh when creating/editing now.

File: src/components/standalone/users_objects/DomainSets.vue [hunk 3]
```diff
@@ -54,6 +57,7 @@ onMounted(() => {
 
 function loadData() {
   listDomainSets()
+  objects.load()
```
Comment (andre8244) [inline src/components/standalone/users_objects/DomainSets.vue:60]: I guess these changes needs to be done in `HostSets.vue` too

Comment (Tbaile) [reply src/components/standalone/users_objects/DomainSets.vue:60]: Whops, forgot to add them to the commit, coming right up

**Review by andre8244 — APPROVED** [review]

**Pull request merged**